src/Form/QuestionType/AbstractQuestionTypeShortAnswer.php

```diff
@@ -56,14 +56,20 @@ public function __construct()
      */
     abstract public function getInputType(): string;
 
+    #[Override]
+    public static function loadJavascriptFiles(): array
+    {
+        return [];
+    }
+
     #[Override]
     public static function formatDefaultValueForDB(mixed $value): ?string
     {
         return $value;
     }
 
     #[Override]
-    public static function validateExtraDataInput(array $input): bool
+    public static function validateExtraDataInput(array &$input): bool
```

Comment:
I am unsure about this for two reasons:
- The method become responsible for validating the data AND fixing the input.
- $input becomes an "Output parameter" (it is modified by the function) which is a bad practice.

I would prefer a dedicated "prepareInput" method that would return the updated input.